timeline-chart/src/layer/time-graph-chart-cursors.ts

```diff
@@ -86,7 +86,7 @@ export class TimeGraphChartCursors extends TimeGraphChartLayer {
         this.unitController.onSelectionRangeChange(() => this.update());
     }
 
-    protected maybeCenterCursor() {
+    public maybeCenterCursor() {
```

Comment:
Could please separate this into a separate patch, so that we can get the time sync merged first?
Then we can tackle the row synchronization.